File: cercle.py
import math
# Demande à l'utilisateur d'entrer le rayon du cercle
rayon = float(input("Entre un rayon :"))

print("ton rayon est : " + str(rayon))

# Calcul de l'aire du cercle
aire = math.pi * rayon * rayon

# Calcul du périmètre du cercle
perimetre = 2 * math.pi * rayon

# Affiche le résultat
print("l'aire est :" + str(aire), "et le perimetre est :" + str(perimetre))

File: cercle.c
#include <stdio.h>
int main(){

	int rayon;
	printf("rayon:");
	scanf("%d",&rayon);
	printf("la valeur du rayon est : %d\n",rayon);
	float pi = 3.14;
	int aire = pi * rayon * rayon;
	int perimetre = 2 * pi * rayon;
	printf("la valeur de l'aire est : %d\n",aire);
	printf("et le périmètre est: %d\n",perimetre);
}
